### PR TITLE
Update GPAC formula to use their new GitHub Repository

### PR DESCRIPTION
**Pull request by spiralman**
Per their blog post: http://gpac.wp.mines-telecom.fr/2015/01/27/leaving-svn-and-sourceforge-using-git-and-github/ they have moved all their source code to GitHub, and also uploaded a new release to their GitHub releases.

### CODE REVIEW

File: Library/Formula/gpac.rb [hunk 1]
```diff
@@ -12,14 +12,11 @@ class Gpac < Formula
   homepage 'http://gpac.wp.mines-telecom.fr/'
 
   stable do
-    url 'https://downloads.sourceforge.net/gpac/gpac-0.5.0.tar.gz'
-    sha1 '48ba16272bfa153abb281ff8ed31b5dddf60cf20'
-
-    # Fixes build against ffmpeg 2.x; backported from upstream SVN
-    patch :DATA
```
Comment (mistydemeo) [inline Library/Formula/gpac.rb]: If you're removing the patch, can you also delete `__END__` and all the lines following it?

Comment (spiralman) [inline Library/Formula/gpac.rb]: Sure thing, sorry I didn't see that at the end. I'll try to send out an
update tomorrow morning CST.

**Comment by spiralman** [comment]
@mistydemeo I made those changes; let me know what you think.

**Comment by mikemcquaid** [comment]
Can you try and add a `depends_on "openssl"`? Thanks!

**Comment by jacknagel** [comment]
ping

**Comment by spiralman** [comment]
Sorry, I had started this for work, but ended up taking a different path. I can try to update it when I get a chance, but I'm not sure when I'll be able to get to it.

**Comment by jacknagel** [comment]
gpac seems to have been updated in the meantime.

**Pull request closed without merging**